source/tokens.rs

```diff
@@ -519,55 +519,148 @@ token!(
     "The `TERNARY_THEN` token.\n\nRepresent the truly block of a ternary condition, e.g. `$x ? … : …`."
 );
 token!(
-    pub THROW: "throw";
+    pub THROW: b"throw";
     "The `THROW` token.\n\nRepresent the throw exception operator, e.g. `throw $e;`."
 );
 token!(
-    pub TRAIT: "trait";
+    pub TRAIT: b"trait";
     "The `TRAIT` token.\n\nRepresent the trait declaration operator, e.g. `trait T { … }`."
 );
 token!(
-    pub TRY: "try";
+    pub TRY: b"try";
     "The `TRY` token.\n\nRepresent the `try` block of a `try`/`catch` control structure, e.g. `try { … } catch (Exception $e) { … }`."
 );
 token!(
-    pub UNSET: "unset";
+    pub UNSET: b"unset";
     "The `UNSET` token.\n\nRepresent the destruction operator, e.g. `unset($x);`."
 );
 token!(
-    pub USE: "use";
+    pub USE: b"use";
     "The `USE` token.\n\nRepresent the importing operator (for namespaces or traits for instance), e.g. `use C\\D;`."
 );
 token!(
-    pub VAR: "var";
+    pub VAR: b"var";
     "The `VAR` token.\n\nRepresent the variable declaration operator (for old PHP versions), e.g. `var $x = …;`."
 );
 token!(
-    pub VARIABLE: "$";
+    pub VARIABLE: b"$";
     "The `VARIABLE` token.\n\nRepresent the variable declaration operator, e.g. `$foo`."
 );
 token!(
-    pub WHILE: "while";
+    pub WHILE: b"while";
     "The `WHILE` token.\n\nRepresent a `while` loop, e.g. `while (…) { … }`."
 );
 token!(
-    pub XOR: "xor";
+    pub XOR: b"xor";
     "The `XOR` token.\n\nRepresent the exclusive disjunction operator, used in a logical expression, e.g. `$x xor $y`."
 );
 token!(
-    pub YIELD: "yield";
+    pub YIELD: b"yield";
     "The `YIELD` token.\n\nRepresent the generator operator, e.g. `yield …;`."
 );
 token!(
-    pub YIELD_FROM: "yield from";
+    pub YIELD_FROM: b"yield from";
```

Comment:
Unrelated to this PR, but the whitespace between `yield` and `from` isn't necessarily a single character.